keylime/cloud_verifier_tornado.py

```diff
@@ -218,23 +219,22 @@ def get(self):
         web_util.echo_json_response(self, 200, "Success", version_info)
 
     def delete(self):
-        web_util.echo_json_response(
-            self, 405, "Not Implemented: Use GET interface instead")
+        web_util.echo_json_response(self, 405, "Not Implemented: Use GET interface instead")
 
     def post(self):
-        web_util.echo_json_response(
-            self, 405, "Not Implemented: Use GET interface instead")
+        web_util.echo_json_response(self, 405, "Not Implemented: Use GET interface instead")
 
     def put(self):
-        web_util.echo_json_response(
-            self, 405, "Not Implemented: Use GET interface instead")
+        web_util.echo_json_response(self, 405, "Not Implemented: Use GET interface instead")
 
     def data_received(self, chunk):
         raise NotImplementedError()
 
 
 class AgentsHandler(BaseHandler):
-    mtls_options = None  # Stores the cert, key and password used by the verifier for mTLS connections
+    mtls_options = (
```

Comment:
Maybe is better to move the comments on top, so black avoid this weird formatting